cmd/config.go

```diff
@@ -19,15 +18,16 @@ var configurationCmd = &cobra.Command{
 	Long: `Modifies Odo specific configuration settings within the config file.
 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false),
+Timeout            - Timeout (in seconds) for openshift server connection check, A value of zero means don't timeout requests`,
```

Comment:
OpenShift

Reply:
Remove the ` A value of zero means don't timeout requests` that doesn't seem needed.

Reply:
Moved that information under Examples